### PR TITLE
Typos in AdminEmailComposeSendAction.java #7141

### LINKED ISSUE #7141: Typos in AdminEmailComposeSendAction.java
Reciver should be corrected to Receiver.

### PR DESCRIPTION
**Pull request by nassimbg**
- Fixed #7141 where I fixed the type "reciver" to "receiver" 
- Changed  class AdminEmailComposeSendAction to a "final" class
- Since its my first pull request, it has the changes of the Introductory  UI enhancement that I proposed in #7146

### CODE REVIEW

**Comment by nassimbg** [comment]
It seems that there was a connection problem while trying to download some artifacts from the maven repo
image

should I repush the build? if yes how can I do it?

**Comment by whipermr5** [comment]
Hi @nassimbg, thanks for your contribution! However, please note that PRs should not contain unrelated changes, as highlighted in our [PR guidelines​](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr). Since #7141 is only about typos, the change of the class to final and the improved about page layout are unrelated changes and should be removed from this PR.

**Comment by whipermr5** [comment]
I notice you've been committing to your master branch - while it's fine for this PR as it's your very first, please follow the [process](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#fixing-issues) for future PRs :)

If you've followed it correctly, your master branch should mirror TEAMMATES' master exactly, and your commits should only appear in your PR branch. An easy way to check this is via GitHub's web UI - there should be a header on the [page of your fork](https://github.com/nassimbg/teammatesBG) that reads `This branch is even with TEAMMATES:master` or `This branch is X commits behind TEAMMATES:master`, but **never** ahead.

**Comment by whipermr5** [comment]
As for restarting the build, the only way is to make some changes and push, or close and reopen the PR.

**Comment by nassimbg** [comment]
@whipermr5 I was trying to get the latest changes from the Teammates master via GitHub Desktop, that's why I've been committing to my master branch. Sorry for that, I think that I should get the latest changes from Teammates master directly to the branch right? and not to my master branch. 
Anyway, I'm still facing a problem with about.jsp file, when I was trying to bring the latest changes, and then I commit to this branch it showed that I have some changes in this file (addition of spaces) is that okay?

**Comment by whipermr5** [comment]
@nassimbg I see; it may be easier to use [SourceTree](https://www.sourcetreeapp.com/) to do some of the things mentioned in our process. The commits to master are fine for now, but you'll still have to find a way to revert the whitespace changes on `about.jsp`. Alternatively you could hard reset your branch to before the first commit and refix the typos, which might actually be less work.

**Comment by nassimbg** [comment]
@whipermr5 the formatting problem has been fixed, it was an auto formatting issue by the IDE since when you submitted the about.jsp file there where additional spaces that should be removed so when I opened this file in my IDE it removed them

File: src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import teammates.common.util.StatusMessageColor;
 import teammates.ui.pagedata.AdminEmailComposePageData;
 
-public class AdminEmailComposeSendAction extends Action {
+public final class AdminEmailComposeSendAction extends Action {
```
Comment (whipermr5) [inline src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java]: Unrelated change

Comment (nassimbg) [inline src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java]: yeah your right, but shouldn't the class be final since it doesn't extend any subclass?

Comment (whipermr5) [inline src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java]: This can be discussed in a separate issue :) It is plausible that this class might be extended in future.
